### PR TITLE
more Twitter link support (vxTwitter / BetterTwitFix, FxTwitter / FixTweet)

### PR DESCRIPTION
**Pull request by jbmagination**

### CODE REVIEW

File: src/modules/processing/hostOverrides.js [hunk 1]
```diff
@@ -3,41 +3,56 @@ export default function (inHost, inURL) {
...
                 host = "twitter";
                 url = url.replace("https://vxtwitter.com/", "https://twitter.com/")
             }
+            if (url.startsWith("https://fixvx.com/")) {
+                host = "twitter";
+                url = url.replace("https://fixvx.com/", "https://twitter.com/")
+            }
+            if (/^https:\/\/(d\.|dl\.|)(fxtwitter|twittpr|fixupx).com\//g.test(url)) {
+                host = "twitter";
+                url = url.replace(url.match(/^https:\/\/(d\.|dl\.|)(fxtwitter|twittpr|fixupx).com\//g)[0], "https://twitter.com/")
```
Comment (wukko) [inline src/modules/processing/hostOverrides.js:33]: nuh uh

**Comment by wukko** [comment]
fxtwitter is intentionally not supported and your implementation is weird

please don't use regex for this

**Comment by jbmagination** [comment]
is there a particular reason why fxtwitter isn't supported?

**Comment by Lenochxd** [comment]
> fxtwitter is intentionally not supported and your implementation is weird
> 
> please don't use regex for this

I was about to PR the same feature, it's true that using regex is a bit weird, but why not accept this kind of links?
It would be enough to replace `fxtwitter.com` etc with `x.com` and it would be simpler. I have a chrome extension that automatically overrides twitter shares with fxtwitter, and support for this domain would be great for users of this extension.

**Comment by wukko** [comment]
i suggest using vxtwitter instead, which is supported by cobalt

**Pull request closed without merging**